kythe/proto/analysis.proto

```diff
@@ -48,10 +48,9 @@ message AnalysisRequest {
 // place.  A given analysis may not produce any outputs.  It is okay for an
 // indexer to send an empty AnalysisOutput message if needed to keep the RPC
 // channel alive; the driver must correctly handle this.
-//
-// The format of value is determined by the analyzer. Kythe language indexers
-// emit wire-format kythe.proto.Entry messages.
 message AnalysisOutput {
+  // The format of `value` is determined by the analyzer. Kythe language
```

Comment:
FWIW Go style eschews use of markup in comments.

Reply:
This isn't Go code, but proto though, so is the Go style relevant?

Reply:
> This isn't Go code, but proto though, so is the Go style relevant?

No, except perhaps as a note for me to read more closely. 😆